student/src/main/java/tds/student/web/handlers/TestShellHandler.java

```diff
@@ -231,31 +236,7 @@ private String parseOutLatencies (Map<String, String> formParams) {
 
     TestOpportunity testOpp = StudentContext.getTestOpportunity ();
 
-    // get responses
```

Comment:
This isn't removed.  @jeffjohnson9046 extracted this logic block to a service since that is probably where this logic should have been in the first place if this application was following a layered architecture..

Replaced here -> https://github.com/SmarterApp/TDS_Student/pull/70/files#diff-06484f1b0d57f1ccd757d75f466f31ffR55